applications/DEMApplication/tests/history_dependent_CLs_tests_files/history_dependent_CLsDEM.mdpa

```diff
@@ -0,0 +1,169 @@
+Begin ModelPartData
+//  VARIABLE_NAME value
+End ModelPartData
+
+Begin Properties 1
+  PARTICLE_DENSITY 1000.0
+  YOUNG_MODULUS 1.0e6
+  POISSON_RATIO 0.2
+  STATIC_FRICTION 0.55
+  DYNAMIC_FRICTION 0.55
+  FRICTION_DECAY 500.0
+  PARTICLE_COHESION 0.0
+  COEFFICIENT_OF_RESTITUTION 0.4
+  PARTICLE_MATERIAL 1
+  ROLLING_FRICTION 0.0
+  ROLLING_FRICTION_WITH_WALLS 0.0
+  CONICAL_DAMAGE_CONTACT_RADIUS 1.0e-5
+  CONICAL_DAMAGE_MAX_STRESS 1.0e6
+  CONICAL_DAMAGE_ALPHA 75.0
+  CONICAL_DAMAGE_GAMMA -0.15
+  LEVEL_OF_FOULING 0.25
+  DEM_DISCONTINUUM_CONSTITUTIVE_LAW_NAME DEM_D_Conical_damage
+  DEM_CONTINUUM_CONSTITUTIVE_LAW_NAME DEMContinuumConstitutiveLaw
+End Properties
+
+Begin Properties 2
+  PARTICLE_DENSITY 1000.0
+  YOUNG_MODULUS 1.0e6
+  POISSON_RATIO 0.2
+  STATIC_FRICTION 0.55
+  DYNAMIC_FRICTION 0.55
+  FRICTION_DECAY 500.0
+  PARTICLE_COHESION 30.0e3
+  INITIAL_COHESION 10.0e3
+  AMOUNT_OF_COHESION_FROM_STRESS 0.2
+  COEFFICIENT_OF_RESTITUTION 0.4
+  PARTICLE_MATERIAL 2
+  DEM_DISCONTINUUM_CONSTITUTIVE_LAW_NAME DEM_D_Stress_Dependent_Cohesive
+  DEM_CONTINUUM_CONSTITUTIVE_LAW_NAME DEMContinuumConstitutiveLaw
+End Properties
```

Comment:
@joaquinirazabal maybe you can also answer this question. Isn't this superfluous and might cause confusion? As this data should be given in the new *.json file now?

Reply:
You have done it right, @KlausBSautter . I will also remove the Properties from mdpa's in this pull request. It's just that I wanted to make sure that old cases did not require to explicitely remove those fields. I could thus make sure that adding the Materials.json was enough.